### DISCUSSION TITLE
Question about reproduce GWB result using geochemistry modules

### DISCUSSION THREAD

**Discussion by MengnanLi91** [idaholab/moose#24988] [Q&A Modules: General]
Hi,

I am working on reproducing a GWB test using geochemistry module but only able to match part of the brine concentrations. I suspect there are something wrong with CO2 solubility calculations but couldn't figure it out. Could you give me some hints on this? I've attached the GWB input, MOOSE input and the corresponding results from both GWB and MOOSE.

Thanks so much for your help!

MOOSE files:
[MOOSE_files.zip](https://github.com/idaholab/moose/files/12089564/MOOSE_files.zip)

GWB files:
[GWB_files.zip](https://github.com/idaholab/moose/files/12089535/GWB_files.zip)

**Comment by loganharbour** [comment]
@WilkAndy or @cpgr; would one of you mind taking a look?

**Reply by cpgr** [reply]
This one might need you @WilkAndy
